<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InventarioDialog</class>
 <widget class="QDialog" name="InventarioDialog">
  <property name="windowTitle">
   <string>Inventario</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing"><number>12</number></property>
   <property name="leftMargin"><number>12</number></property>
   <property name="topMargin"><number>12</number></property>
   <property name="rightMargin"><number>12</number></property>
   <property name="bottomMargin"><number>12</number></property>
   <item>
    <layout class="QGridLayout" name="layoutInputs">
     <property name="spacing"><number>12</number></property>
     <property name="leftMargin"><number>12</number></property>
     <property name="topMargin"><number>12</number></property>
     <property name="rightMargin"><number>12</number></property>
     <property name="bottomMargin"><number>12</number></property>
     <item row="0" column="0">
      <widget class="QLineEdit" name="lineEditSKU">
       <property name="placeholderText"><string>SKU</string></property>
       <property name="accessibleName"><string>sku</string></property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditNombre">
       <property name="placeholderText"><string>Nombre del producto</string></property>
       <property name="accessibleName"><string>nombre_producto</string></property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QComboBox" name="comboCategoria">
       <property name="editable"><bool>true</bool></property>
       <property name="accessibleName"><string>categoria</string></property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QSpinBox" name="spinCantidad">
       <property name="minimum"><number>0</number></property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QSpinBox" name="spinStockMin">
       <property name="minimum"><number>0</number></property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QDoubleSpinBox" name="doubleCosto">
       <property name="decimals"><number>2</number></property>
       <property name="minimum"><double>0.0</double></property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="doublePrecio">
       <property name="decimals"><number>2</number></property>
       <property name="minimum"><double>0.0</double></property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="lineUbicacion">
       <property name="placeholderText"><string>Ubicaci√≥n</string></property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="lineProveedor">
       <property name="placeholderText"><string>Proveedor</string></property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="btnAgregar">
       <property name="text">
        <string>Agregar</string>
       </property>
       <property name="default"><bool>true</bool></property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableProductos">
     <property name="alternatingRowColors"><bool>true</bool></property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="selectionMode"><enum>QAbstractItemView::SingleSelection</enum></property>
     <property name="sortingEnabled"><bool>true</bool></property>
     <attribute name="horizontalHeaderStretchLastSection"><bool>true</bool></attribute>
    </widget>
   </item>
  <item>
   <layout class="QHBoxLayout" name="layoutButtons">
     <item>
      <widget class="QPushButton" name="btnEliminar">
       <property name="text">
        <string>Eliminar</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnCerrar">
       <property name="text">
        <string>Cerrar</string>
       </property>
      </widget>
     </item>
    </layout>
  </item>
   </layout>
  </widget>
  <tabstops>
   <tabstop>lineEditSKU</tabstop>
   <tabstop>lineEditNombre</tabstop>
   <tabstop>comboCategoria</tabstop>
   <tabstop>spinCantidad</tabstop>
   <tabstop>spinStockMin</tabstop>
   <tabstop>doubleCosto</tabstop>
   <tabstop>doublePrecio</tabstop>
   <tabstop>lineUbicacion</tabstop>
   <tabstop>lineProveedor</tabstop>
   <tabstop>btnAgregar</tabstop>
   <tabstop>tableProductos</tabstop>
   <tabstop>btnEliminar</tabstop>
   <tabstop>btnCerrar</tabstop>
  </tabstops>
  <resources/>
  <connections/>
 </ui>
